docs/src/api-testing-python.md

```diff
@@ -0,0 +1,289 @@
+---
+id: api-testing
+title: "API testing"
+---
+
+Playwright can be used to get access to the [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) API of
```

Comment:
Can you merge this with api-testing-java.md to avoid duplication or it's easier to keep them separate?

Reply:
I think easier to separate, since in almost every paragraph there are small differences compared to Java. (yield, pytest specific, afterAll etc.)